modules/odc-creating-apiserversource.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * applications/application_life_cycle_management/odc-knative-event-sources.adoc
+
+[id="odc-creating-apiserversource_{context}"]
+= Creating an ApiServerSource event source using the Developer perspective
+
+.Procedure
+
+. Navigate to the **Add** page.
++
+image::event-sources-odc.png[Add an Event Source in the Developer perspective]
+. Select **Event Sources** -> **ApiServerSource**.
++
+image::apiserversource-odc.png[Create an ApiServerSource]
+. Configure the **ApiServerSource** settings:
++
+image::apiserver-odc-config-1.png[Configuring the ApiServerSource]
++
+.. Enter `sources.knative.dev/v1alpha1` as the **APIVERSION**, and `ApiServerSource` as the **KIND**.
+// TODO: make sure version is right across all serverless docs, if this needs to be updated, update in `kn` docs too.
+// .. What does 'mode' do?
+.. Select the **Service Account Name** for the service account that you created for event sources.
+.. Select the targeted Knative service from the dropdown menu in **Sink** -> **Knative Service**.
+// .. Add details about the image:
+// General -> Application Name, Name
+// image::apiserver-odc-config-2.png[Configuring the ApiServerSource]
+. Click **Create**.
+
+.Verification steps
+
+. After you have created the ApiServerSource, you will see it connected to the service it is sinked to in the **Topology** view.
++
+image::toplogy-odc-apiserver.png[ApiServerSource Topology view]
+
+// . Create events by launching a pod in the default namespace.
+// TODO: How to do this? What should a user see?
+
+= Deleting the ApiServerSource
```

Comment:
EventSources can be deleted on right click via context menu or on click it'll open side there are option menu with which we can delete.

<img width="803" alt="Screenshot 2020-06-16 at 12 14 27 PM" src="https://user-images.githubusercontent.com/5129024/84740604-535c4800-afcb-11ea-91ee-a16950f130a8.png">